com.obsproject.Studio.Plugin.ScaleToSound.yaml

```diff
@@ -0,0 +1,25 @@
+id: com.obsproject.Studio.Plugin.ScaleToSound
+branch: stable
+runtime: com.obsproject.Studio
+runtime-version: stable
+sdk: org.kde.Sdk//5.15-21.08
+build-extension: true
+separate-locales: false
+appstream-compose: false
+build-options:
+  prefix: /app/plugins/ScaleToSound
+modules:
+  - name: scale-to-sound
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DBUILD_STANDALONE=On
+    post-install:
+      - install -Dm644 --target-directory=${FLATPAK_DEST}/share/metainfo ${FLATPAK_ID}.metainfo.xml
+      - appstream-compose --basename=${FLATPAK_ID} --prefix=${FLATPAK_DEST} --origin=flatpak ${FLATPAK_ID}
+    sources:
+      - type: git
+        url: https://github.com/Qufyy/obs-scale-to-sound.git
+        commit: b027f79fd7b8720429c745704aae9f6c474eca21
```

Comment:
why not using the release tag and commit?

actual releae are prefered.

Reply:
I updated the CMakeLists.txt so it can be built outside of the OBS build process after the latest actual release

Reply:
fair enough